audbackend/core/api.py

```diff
@@ -115,6 +117,54 @@ def create(
     return backends[name][host][repository]
 
 
+def delete(
+        name: str,
+        host: str,
+        repository: str,
+):
+    r"""Delete repository and remove backend instance.
+
+    .. warning:: Deletes the repository and all its content.
+
+    If an instance of the backend exists,
+    it will be removed from the available instances.
+    See also :func:`audbackend.available`.
+
+    Args:
+        name: alias under which backend class is registered
+        host: host address
+        repository: repository name
+
+    Raises:
+        BackendError: if an error is raised on the backend
+        ValueError: if no backend class with alias ``name``
+            has been registered
+
+    Examples:
+        >>> create(
+        ...     'file-system',
+        ...     'host',
+        ...     'doctest',
+        ... ).ls()
+        [('a.zip', '1.0.0'), ('a/b.ext', '1.0.0'), ('name.ext', '1.0.0'), ('name.ext', '2.0.0')]
+        >>> delete(
+        ...     'file-system',
+        ...     'host',
+        ...     'doctest',
+        ... )
```

Comment:
This feels slightly unsymmetrical. `create()` does not (always) create a new repo, but just an instance to access the repo, whereas `delete()` really deletes the repo.
But I also don't have an idea how to improve this.

Reply:
Right, `create()` only creates a new repo if it does not exist yet. Maybe `audbackend.open()` would be a better fit?

Reply:
or maybe `init()`?

Reply:
I created https://github.com/audeering/audbackend/issues/99 to handle this outside of this pull request.